src/Generator/FileGenerator.php

```diff
@@ -219,30 +231,29 @@ public function getUses($withResolvedAs = false)
     }
 
     /**
-     * @param  array $uses
+     * @param InputUses $uses
      * @return FileGenerator
      */
     public function setUses(array $uses)
     {
         foreach ($uses as $use) {
             $use = (array) $use;
             if (array_key_exists('use', $use) && array_key_exists('as', $use)) {
-                $import = $use['use'];
-                $alias  = $use['as'];
-            } elseif (count($use) == 2) {
+                $this->setUse($use['use'], $use['as']);
+            } elseif (count($use) === 2) {
                 [$import, $alias] = $use;
+
+                $this->setUse($import, $alias);
```

Comment:
```suggestion
                $this->setUse(...$use);
```

You should be able to spread the `$use` variable directly into the method parameter list.

Reply:
I'd avoid this, since the structure may be in worse shape than what we think here.

While I am for improvements, unless the types become much stricter, it is a risk to allow splatting anything in here, especially if `->setUse()` changes signature in future.